torchao/experimental/tests/test_int8_dynamic_activation_intx_weight.py

```diff
@@ -216,6 +216,53 @@ def test_export_compile_aoti_PackedLinearInt8DynamicActivationIntxWeightLayout(
             aoti_results = fn(activations)
             self.assertTrue(torch.allclose(eager_results, aoti_results))
 
+    def test_export_dynamic_shape_PackedLinearInt8DynamicActivationIntxWeightLayout(
+        self,
+    ):
+        granularity = PerRow()
+        m = 3
+        k0 = 512
+        k1 = 256
+        k2 = 128
+        k3 = 1024
+        weight_dtype = torch.int4
```

Comment:
Maybe would be good to loop through and test all of the possible granularities from 2-7